lib/routers/establishment/index.js

```diff
@@ -117,6 +117,7 @@ router.put('/:establishment/grant',
   submit('grant')
 );
 
+router.use('/:establishment/pil-reviews', require('./pil-reviews'));
```

Comment:
Could/should this be scoped under the pils router?

Reply:
Although the fact that the "pils" router is nested under "profiles" makes it arguably a hack either way.

Reply:
It's possible we want to split the establishment-level PIL listing functions and the profile level PIL application functions.

Reply:
we have /establishment/pils, but if I route this under there it hits /est/pils/:pilId, so need to add some horrible code like 
```
if (req.params.pilId === 'reviews') {
  return next('route');
}
```

Reply:
I think it's _probably_ better to split the routers as I don't think we use `/est/pils/:id`, so have a separate PIL router with GET / - list and GET /reviews - this stuff?

Reply:
I agree.